Sources/PubNubChat/PubNub/Models/ChatMessage.swift

```diff
@@ -37,25 +37,25 @@ public struct ChatMessage<CustomData: ChatCustomData>: Identifiable, Codable {
   
   public struct Content: JSONCodable {
     public var id: String
-    
-    public var dateCreated: Date
-    
-    public var contentType: String
-    public var contentPayload: MessageContent
-    
-    public var custom: CustomData.Message
+    public var text: String
+    public var contentType: String?
+    public var content: MessageContent?
+    public var custom: CustomData.Message?
+    public var createdAt: Date
     
     public init(
       id: String = UUID().uuidString,
-      dateCreated: Date = Date(),
-      content: MessageContent,
-      custom: CustomData.Message
+      text: String = String(),
+      content: MessageContent? = nil,
+      custom: CustomData.Message? = nil,
```

Comment:
Can this return to a non-optional? 

See comment on dynamic member lookup method for reason